vite.config.js

```diff
@@ -0,0 +1,19 @@
+// vite.config.js
+import { resolve } from "path";
+import { defineConfig } from "vite";
+import { nodePolyfills } from "vite-plugin-node-polyfills";
+
+export default defineConfig({
+  build: {
+    lib: {
+      // Could also be a dictionary or array of multiple entry points
+      entry: resolve(__dirname, "index.js"),
+      name: "html-to-rtf",
+      // the proper extensions will be added
+      fileName: "html-to-rtf",
+    },
+    rollupOptions: {},
+  },
+
+  plugins: [nodePolyfills()],
```

Comment:
We'll node-polyfill here so we don't have to do that across all of kong-fu, which does not need Node APIs polyfilled.